Ready for media-center.io skill

m0ngr31 opened the pull request.

m0ngr31 committed:
Ready for media-center.io skill

jingai commented:
Looks fine -- I'll test it this weekend to make sure it doesn't break anything for us self-hosters :)

jingai committed:
Fix intents that call other intents.

The @preflight_check decorator prevents us from abusing this, and we
probably shouldn't have been anyway.

m0ngr31 committed:
Merge pull request #236 from jingai/media-center-io-fix-double-calls

Fix intents that call other intents.
